digital-workspace/latest/config/index.md

```diff
@@ -73,8 +73,14 @@ Setting permissions for folders and files allows you to manage what type of acce
 
 4. (Optional) Use the slider to allow the added users or groups to inherit the permissions from the parent folder.
 
-## Search Configuration
+## Search
```

Comment:
I would mention the difference between filters and facets - it also matters when admin want to change the configuration of Advanced Search.
Filters are visible in Advanced search from the beginning (when there are no search results yet) and the options to select can be defined in the config file.
Facets appear only when they have any options to select and the options are populated based on the search results. The example of this kind of Facet in "People" object, which allows to filter by creator or modifier. If there are no search results, the facets are not visible. 
Admin can configure both types: filters and facets.

Reply:
Filters - they are visible from the beginning, before any search has been done (or if there are no search results at all). The visibility of filters and options to select inside them DOES NOT depend on the search results. The options to select are preconfigured in configuration files.
Facets - the opposite: They are NOT visible from the beginning of the searching. Their appearance depends on the search results, because the options to select are populated based on the found nodes after search query.
For example:
user clicks magnifying glass icon to start searching and there are only filters visible from the beginning
user do some filtering or searching and got some search results
the "Creator" facet appears (because the nodes have a creator) with the creators list of the found nodes on the list in options
if none of node was ever edited and they have no "modifiers", the "Modifier" facet won't appear at all (there is nothing to filter by)

Reply:
Aritri will add the configuration part to the documentation once the config has been documented. See https://alfresco.atlassian.net/browse/PRODENG-230